File: newbiest-client/src/js/dataModel/entitylist/EntityListRequestBody.js
class EntityListRequestBody {

    entityModel;
    whereClause;
    orderBy;
    maxResult;
    firstResult;
    fields;

    constructor(entityModel, whereClause, orderBy, maxResult, firstResult, fields) {
        this.entityModel = entityModel;
        this.whereClause = whereClause;
        this.orderBy = orderBy;
        this.maxResult = maxResult;
        this.firstResult = firstResult;
        this.fields = fields;
    }

    setEntityModel(entityModel) {
        this.entityModel = entityModel;
    }
}

export {EntityListRequestBody}

File: newbiest-framework/src/main/java/com/newbiest/base/rest/entityList/EntityListRequestBody.java
package com.newbiest.base.rest.entityList;

import com.newbiest.msg.RequestBody;
import lombok.Data;

import java.util.List;

/**
 * Created by guoxunbo on 2018/7/31.
 */
@Data
public class EntityListRequestBody extends RequestBody {

    private static final long serialVersionUID = 1L;

    private String entityModel;

    private String whereClause;

    private String orderBy;

    private int maxResult;

    private int firstResult;

    private List<String> fields;
}
